File: php/Loggly.php
<?php
include("../IronWorker.class.php");

$name = "loggly-php";

$iw = new IronWorker('config.ini');

# Creating and uploading code package.
$iw->upload(dirname(__FILE__)."/workers/loggly", 'loggly.php', $name);

for ($i = 1; $i <= 50; $i++)
{
    $payload = array('api_key' => LOGGLY_KEY, 'i' => $i);

    $iw->postTask($name, $payload);
}

File: python/Loggly.py
from iron_worker import *

worker = IronWorker()

package = CodePackage()
package.name = "LogglyExample"
package.merge("loggly", ignoreRootDir=True)
package.merge_dependency("hoover")
package.merge_dependency("httplib2")
package.executable = "loggly.py"

worker.upload(package)
payload = {'loggly': {'subdomain': 'LOGGLY_SUBDOMAIN', 'username': 'LOGGLY_USERNAME', 'password': 'LOGGLY_PASSWORD'}}

task = worker.queue(code_name='LogglyExample', payload=payload)
print task
